src/components/ebay-notice/README.md

```diff
@@ -19,3 +23,11 @@ Name | Type | Stateful | Description
 `status`  | String | No | "priority" (default), "confirmation" or "information"
 `aria-text` | String | No | adding description for the notice for a11y users
 `heading-level` | String | No| used in case of "page" level notices to specify the heading tag according to the notice's placement
+`dismissible` | Boolean | No | used to specify the dismiss button in page notice
+`hidden` | Boolean | Yes | whether the widget is hidden or not (Note: not supported as initial attribute)
+
+### ebay-notice Events
+
+Event | Description
+--- | ---
+`notice-change` | the page notice was dismissed/undismissed
```

Comment:
A `change` event would need to pass data along too, or alternatively maybe we should have 2 events `notice-hide` and `notice-show` similar to the Menu.

Reply:
done